doc/sections/options.md

```diff
@@ -51,7 +51,7 @@ When in default mode, the following flags are available for configuration:
 
 #### Confluence checking
 
-Confluence checking (and also termination) must be established for each of the
+Confluence (and also termination) must be established for each of the
 considered rewriting systems contained in `lambdapi` files. By default,  these
```

Comment:
each ... system-s
must -> Lambdapi provides an option to check the confluence of the set of rewriting rules declared in a file.

Question: what is this set? Is it the smallest set obtained by adding all the rules in the file and its dependencies? What if several file names are given in argument?

Reply:
The set of rewrite rules considered is the set of all rules declared in any of the file of the transitive closure computed via the `require`.
I don't understand what you mean by : 
> must -> Lambdapi provides an option to check the confluence of the set of rewriting rules declared in a file by calling external provers.
Your suggestion seems to be a reformulation of the next sentence. So does it mean "suppress the sentence containing must and replace the next one by"?